components/Home/sidebar/HomeRightSidebar.tsx

```diff
@@ -16,23 +15,30 @@ export default function HomeRightSidebar(): ReactElement {
       <ColumnContainer overrideStyles={styles.HomeRightSidebarContainer}>
         <ExitableBanner
           bannerKey={"$rhRightSidebarInfoCarousel"}
-          content={<Carousel carouselItems={[<CarouselItem />]} />}
+          content={
+            <RhCarousel rhCarouselItemProps={[{ title: "", body: "yoyo" }]} />
```

Comment:
😄 I imagine this will be changed in future PR

Reply:
haha yes.